test/mappers/session/dissolutionSession.mapper.test.ts

```diff
@@ -54,4 +55,23 @@ describe('DissolutionApprovalMapper', () => {
       assert.equal(result.applicant, 'some signatory name')
     })
   })
+
+  describe('mapToDissolutionConfirmation', () => {
+    let dissolution: DissolutionGetResponse
+
+    const BUCKET: string = 'some-bucket'
+    const KEY: string = 'some-key'
+
+    beforeEach(() => dissolution = generateDissolutionGetResponse())
+
+    it('should map the certificate fields to a confirmation session', () => {
+      dissolution.certificate_bucket = BUCKET
+      dissolution.certificate_key = KEY
+
+      const result: DissolutionConfirmation = mapper.mapToDissolutionConfirmation(dissolution)
+
+      assert.equal(result.certificateKey, KEY)
+      assert.equal(result.certificateBucket, BUCKET)
+    })
+  })
 })
```

Comment:
New line here also?

Reply:
Done - also added rule to tslint so had to fix all violations in the app